lib/line-protocol.js

```diff
@@ -73,10 +73,10 @@ internals.values = {
         {},
         internals.formatError(event.error),
         {
-            id     : internals.String(event.id),
-            url    : internals.String(event.url && Url.format(event.url)),
-            method : internals.String(event.method && event.method.toUpperCase()),
-            tags   : internals.String(event.tags)
+            id: internals.String(event.id),
```

Comment:
I _used_ to line them up, but since I've been using `standard` in some projects, I just leave them alone. I don't really have a strong opinion either, however, a constant style should be used throughout the package.